app/src/main/kotlin/com/wire/android/ui/authentication/welcome/Welcome.kt

```diff
@@ -80,7 +83,7 @@ private fun WelcomeContent(navController: NavController) {
                 WelcomeCarousel()
             }
 
-            Column(modifier = Modifier.padding(top = 40.dp, bottom = 52.dp, start = 16.dp, end = 16.dp),) {
+            Column(modifier = Modifier.padding(top = 40.dp, bottom = 52.dp, start = 16.dp, end = 16.dp)) {
```

Comment:
why not using `dimensions` instead of `.dp`?

Reply:
🚀  This view was created before the final changes to Dimensions, but you're right - I updated it